spring-rabbit/src/main/java/org/springframework/amqp/rabbit/connection/RabbitConnectionFactoryBean.java

```diff
@@ -125,7 +125,7 @@
 
 	private String trustStoreType;
 
-	private String sslAlgorithm = TLS_V1_1;
+	private String sslAlgorithm = TLS_V1_2;
```

Comment:
We should simply remove the default; and remove the `sslAlgorithmSet` boolean; then use `useSSlProtocol()` if `this.sslAgorithm` is not null.

https://github.com/spring-projects/spring-amqp/blob/f7b7bedefc3de6149d5d1222c3c30d6eac40d6e2/spring-rabbit/src/main/java/org/springframework/amqp/rabbit/connection/RabbitConnectionFactoryBean.java#L732-L737

However, unfortunately, we can't merge this until we switch the master branch to 2.3.